synapse/storage/data_stores/main/devices.py

```diff
@@ -725,6 +720,21 @@ def __init__(self, database: Database, db_conn, hs):
             BG_UPDATE_REMOVE_DUP_OUTBOUND_POKES, self._remove_duplicate_outbound_pokes,
         )
 
+        # create a unique index on device_lists_outbound_last_success
+        self.db.updates.register_background_index_update(
+            "device_lists_outbound_last_success_unique_idx",
+            index_name="device_lists_outbound_last_success_unique_idx",
+            table="device_lists_outbound_last_success",
+            columns=["destination", "user_id"],
+            unique=True,
+        )
+
+        # once that completes, we can remove the old non-unique indexes.
```

Comment:
```suggestion
        # once that completes, we can remove the old non-unique index.
```

Looks like we're only dropping one index.